common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -3879,24 +3879,32 @@ schematic = (function() {
 	////////////////////////////////////////////////////////////////////////////////
 
 	Schematic.prototype.add_tool = function(icon,tip,callback) {
-	    var tool;
+	    var tool, child, label;
+
+        tool = document.createElement('button');
+        child = document.createElement('img');
+        label = document.createElement('span');
+        
+        label.innerHTML = tip;
+        label.classList.add('sr');
+        tool.appendChild(label);
+
 	    if (icon.search('data:image') != -1) {
-		tool = document.createElement('img');
-		tool.src = icon;
+            tool.style.backgroundImage = 'none';
+            child.setAttribute('src', icon);
+            child.setAttribute('alt', tip);
+            tool.appendChild(child);
 	    } else {
-		tool = document.createElement('span');
-		tool.style.font = 'small-caps small sans-serif';
-		var label = document.createTextNode(icon);
-		tool.appendChild(label);
+    		tool.style.font = 'small-caps small sans-serif';
```

Comment:
It should be possible to make indentation consistent within the parts you've changed.  These four lines should be at the same level at least.